src/content/changelog/4-64-0-2020-05-04.md

```diff
@@ -0,0 +1,35 @@
+---
+title: '4.64.0: 2020-05-04'
+date: 2020-05-04T04:00:00.000Z
+version: 4.64.0
+changelog:
+  - API
+---
+
+### Added
+
+- Validation improvements were added to prevent Domain Record naming conflicts.
+
+  - If you attempt to create ([POST /domains/{domainId}/records](/api/v4/domains-domain-id-records/#post)) or update ([PUT /domains/{domainId}/records/{recordId}](/api/v4/domains-domain-id-records-record-id/#put)) a Domain Record of `type` `CNAME`, `A`, `AAA`, or `MX` with a `name` value that already exists, you will receive an error message indicating that the name already exists.
```

Comment:
this change was reverted @leslitagordita

Reply:
ahh, okay, will remove this entry!